internal/provider/harness.go

```diff
@@ -57,14 +62,11 @@ func (r *HarnessResource) ModifyPlan(ctx context.Context, req resource.ModifyPla
 
 	inv := InventoryDataSourceModel{}
 	if diags := req.Config.GetAttribute(ctx, path.Root("inventory"), &inv); diags.HasError() {
-		resp.Diagnostics.AddError("failed to add harness", "retrieving inventory")
```

Comment:
Forgive my ignorance, what's the purpose of removing these diagnostic errors here?

Reply:
They were swallowing the actual error outputs 😬